tests/testthat/outputs/vars-select-oob-errors.txt

```diff
@@ -19,7 +19,7 @@ x Can't subset element with unknown name `foo`.
 > # Positions
 > vars_select(letters, 30, 50, 100)
 Error: Must select existing columns.
-x Can't subset positions 30, 50 and 100.
+x Can't subset position 100.
```

Comment:
Was this a deliberate change?

Reply:
Yes but I found a way to get the original message back. Early validation is only needed for the RHS of set-difference (as OOB values might be filtered out). We now validate lazily for other operations to get a more complete error report.